examples/multimodal/graphs/agg.py

```diff
@@ -16,7 +16,8 @@
 
 from components.decode_worker import VllmDecodeWorker
 from components.encode_worker import VllmEncodeWorker
-from components.frontend import Frontend
 from components.processor import Processor
 
+from components.frontend import Frontend
```

Comment:
Looks like for this change it doesn't change anything other than move the import to the other line. Is this required?

Reply:
The `pre-commit` hook insisted. :shrug: